migrations/versions/5dcb69824a56_.py

```diff
@@ -0,0 +1,48 @@
+"""empty message
+
+Revision ID: 5dcb69824a56
+Revises: af9215895b80
+Create Date: 2017-06-16 14:35:52.300308
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5dcb69824a56'
+down_revision = 'af9215895b80'
```

Comment:
Change down revision to `c79e6d36a194` from https://github.com/fossasia/open-event-orga-server/commit/65522599d3b9f0f82e92f8b4bcc97fa938fcaaba, so that we do not have multiple heads.
Rest all looks good